tests/foreman/ui/test_bookmarks.py

```diff
@@ -40,15 +94,21 @@ def test_positive_create_bookmark_populate_auto(self):
 
         @Assert: No errors, Bookmark is displayed, controller matches the
         entity the bookmark was created for
-
-        @Status: Manual
         """
-        for entity in BOOKMARK_ENTITIES:
-            if entity.get('setup'):
-                create_entity(entity['setup'])
-            pass
+        for entity in self.entities:
+            with self.subTest(entity):
+                with Session(self.browser) as session:
```

Comment:
Good question. We have 45 entities, let's assume we navigate 15-20 different pages per entity. So it's about 675-900 pages in one session. I'm not sure if it's a good idea to do that in one browser session. Remember from our previous experience, when we were executing a lot of tests in a single browser instance and at some point browser just stopped responding? I'm just a bit afraid we can face something similar. But you're more experienced in this than I, so please decide :)